index.js

```diff
@@ -5,4 +5,11 @@ var
 // Adds on the fly methods promisification
 Kuzzle.prototype.bluebird = bluebird;
 
+if (typeof window !== 'undefined' && typeof BUILT === 'undefined') {
+  throw new Error('It looks like you are using the Nodejs version of Kuzzle SDK ' +
+               'in a browser. ' +
+               'It is strongly recommended to use the browser-specific build instead. ' +
+               'Learn more at https://github.com/kuzzleio/sdk-javascript/tree/master#browser');
```

Comment:
I cannot find that anchor in our README file, either in `master`, `5.x` or `6.x`

Reply:
the anchor in the README is `#javascript` 
should rather be `#browser`, imho, so the link here is correct, bug we need to fix the Readme.

Reply:
@scottinet https://github.com/kuzzleio/sdk-javascript/tree/5.x#browser